### PR TITLE
Update new-distro.md

### PR DESCRIPTION
**Pull request by sophieyfang**
## Description

Update new-distro.md

## Checklist:
- Unit tests
  - [x] Unit tests do not apply.
  - [ ] Unit tests have been added/modified and passed for this PR.
- Integration tests
  - [x] Integration tests do not apply.
  - [ ] Integration tests have been added/modified and passed for this PR.
- Documentation
  - [x] This PR introduces no user visible changes.
  - [ ] This PR introduces user visible changes and the corresponding documentation change has been made.
- Minor version bump
  - [x] This PR introduces no new features.
  - [ ] This PR introduces new features, and there is a separate PR to bump the [minor version](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/VERSION) since the last [release](https://github.com/GoogleCloudPlatform/ops-agent/releases) already.
  - [ ] This PR bumps the version.

### CODE REVIEW

File: dev-docs/new-distro.md [hunk 1]
```diff
@@ -39,10 +39,12 @@ manifest as runtime errors that won't show up until tests are run.
...
 1.  Temporarily repurpose one of the existing Kokoro builds for testing
     your new distro. For example, in
-    [build/presubmit/bullseye.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/build/presubmit/bullseye.gcl),
+    [kokoro/config/build/presubmit/bullseye_x86_64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/build/presubmit/bullseye_x86_64.gcl)
+    or [kokoro/config/build/presubmit/bullseye_aarch64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/build/presubmit/bullseye_aarch64.gcl),
     replace `bullseye` with `$DISTRO_SHORT` and change `deb` to `rpm` if
     needed. Then in
-    [test/ops_agent/bullseye.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/ops_agent/bullseye.gcl),
+    [kokoro/config/test/ops_agent/bullseye_x86_64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/ops_agent/bullseye_x86_64.gcl)
+    or [kokoro/config/test/ops_agent/bullseye_aarch64.gcl](https://github.com/GoogleCloudPlatform/ops-agent/blob/master/kokoro/config/test/ops_agent/bullseye_aarch64.gcl),
```
Comment (martijnvans) [inline dev-docs/new-distro.md:47]: Maybe also update line 61 to say: `bullseye_$arch.gcl`

Comment (sophieyfang) [reply dev-docs/new-distro.md:47]: Done.

**Review by martijnvans — APPROVED** [review]

File: dev-docs/new-distro.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ for the new distro. You can duplicate the most recent distro as a starting
 point. Then iterate on your change to `compile.go` by running:
 
 ```shell
-go run dockerfiles/compile.go && docker build --target $DISTRO_SHORT -t scratch .
+DOCKER_BUILDKIT=1 go run dockerfiles/compile.go && docker build --target $DISTRO_SHORT -t scratch .
```
Comment (martijnvans) [inline dev-docs/new-distro.md:29]: wait, I just noticed that DOCKER_BUILDKIT=1 should be attached to the docker build part of the command, not the go run part.

**Review by martijnvans — CHANGES_REQUESTED** [review]
oops, see my comment about DOCKER_BUILDKIT=1 above

File: dev-docs/new-distro.md (hunk 2 above)
Comment (sophieyfang) [reply dev-docs/new-distro.md]: 😛 done

**Review by martijnvans — APPROVED** [review]

**Pull request merged**